src/main/resources/application.yml

```
@@ -17,3 +17,9 @@ management:
     health:
       show-details: always
       show-components: always
+
+braintree:
+  public-key: ypfct5b4m3hvx7yv
+  private-key: 39ec1ac885710525f99900a314a889c0
+  merchant-id: zkfjnzn48z965sv5
```

à remplacer par des valeurs du secret manager
```suggestion
  public-key: ${sm://BRAINTREE_PUBLIC_KEY}
  private-key: ${sm://BRAINTREE_PRIVATE_KEY}
  merchant-id: ${sm://BRAINTREE_MERCHANT_ID}
```